source/includes/mongodb-compatibility-table-node.rst

```diff
@@ -17,7 +17,21 @@
      - MongoDB 3.0
      - MongoDB 2.6
 
-   * - 6.0 to 6.9
+   * - 6.9
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓ [#deprecated]_
```

Comment:
S: now that you changed the footnote name, update here as well

Reply:
d'oh